samples/betaFeatures.js

```diff
@@ -0,0 +1,586 @@
+        .map(
+          result =>
+            ` Channel Tag: ` +
+            result.channelTag +
+            ` ` +
+            result.alternatives[0].transcript
+        )
+        .join('\n');
+      console.log(`Transcription: \n${transcription}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speechTranscribeMultichannelGcs]
+}
+
+// # MULTI LANGUAGE
+
+function speechTranscribeMultilang(speechFile) {
+  //[START speechTranscribeMultilang]
+
+  const fs = require('fs');
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  const config = {
+    encoding: 'LINEAR16',
+    sampleRateHertz: 44100,
+    languageCode: `en-US`,
+    alternativeLanguageCodes: [`es-ES`, `en-US`],
+    audioChannelCount: 2,
+  };
+
+  const audio = {
+    content: fs.readFileSync(speechFile).toString('base64'),
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      console.log(`Transcription: ${transcription}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+
+  //[END speechTranscribeMultilang]
+}
+
+function speechTranscribeMultilangGCS(gcsUri) {
+  // [START speechTranscribeMultilangGCS]
+
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  const config = {
+    encoding: 'LINEAR16',
+    sampleRateHertz: 44100,
+    languageCode: `en-US`,
+    alternativeLanguageCodes: [`es-ES`, `en-US`],
+    audioChannelCount: 2,
+  };
+
+  const audio = {
+    uri: gcsUri,
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .longRunningRecognize(request)
+    .then(data => {
+      const operation = data[0];
+      // Get a Promise representation of the final result of the job
+      return operation.promise();
+    })
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      console.log(`Transcription: ${transcription}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speechTranscribeMultilangGCS]
+}
+
+// WORD LEVEL CONFIDENCE
+function speechTranscribeWordLevelConfidence(speech_file) {
+  // [START speechTranscribeWordLevelConfidence]
+  const fs = require('fs');
+
+  // Imports the Google Cloud client library
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  const config = {
+    encoding: `FLAC`,
+    sampleRateHertz: 16000,
+    languageCode: `en-US`,
+    enableWordTimeOffsets: true,
```

Comment:
Do you need to enableWordTimeOffsets in order to get word confidence? If not, I'd prefer omitting it.

Reply:
no, fixed